### PR TITLE
additional blacklisting

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

File: etc/disable-programs.inc [hunk 1]
```diff
@@ -538,6 +538,7 @@ blacklist ${HOME}/.w3m
 blacklist ${HOME}/.warzone2100-3.*
 blacklist ${HOME}/.waterfox
 blacklist ${HOME}/.weechat
+blacklist ${HOME}/.wget-hsts
```
Comment (SkewedZeppelin) [inline etc/disable-programs.inc]: this will need to be noblacklisted in wget.profile

File: etc/disable-common.inc [hunk 2]
```diff
@@ -11,12 +11,15 @@ blacklist ${HOME}/.local/share/Trash
...
 blacklist-nolog ${HOME}/.adobe
 blacklist-nolog ${HOME}/.cache/greenclip*
+blacklist-nolog ${HOME}/.histfile
 blacklist-nolog ${HOME}/.history
 blacklist-nolog ${HOME}/.kde/share/apps/klipper
 blacklist-nolog ${HOME}/.kde4/share/apps/klipper
 blacklist-nolog ${HOME}/.local/share/fish/fish_history
 blacklist-nolog ${HOME}/.local/share/klipper
 blacklist-nolog ${HOME}/.macromedia
+blacklist-nolog ${HOME}/.python-history
```
Comment (SkewedZeppelin) [inline etc/disable-common.inc]: these should probably be noblacklisted in pycharm-*.profile

**Comment by SkewedZeppelin** [comment]
@rusty-snake If you are good with this I will merge

**Comment by rusty-snake** [comment]
@SkewedZeppelin I'm good.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**